catalog-info.yaml

```diff
@@ -40,5 +40,5 @@ metadata:
 spec:
   type: frontend
   lifecycle: alpha
-  owner: "group:default/fcp-sfd"
+  owner: "group:default/fpc-sfd"
```

Comment:
Is this correct? `fpc` ?

Reply:
It is definitely not correct... will change this now, thank you

Reply:
Done